todo_app/tests/.env.test

```diff
@@ -8,10 +8,16 @@ FLASK_DEBUG=true
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
 
-TRELLO_BOARD_ID='fake_board_id'
-TRELLO_APIKEY='fake_apikey'
-TRELLO_TOKEN='fake_trello_token' 
+# TRELLO_BOARD_ID='fake_board_id'
+# TRELLO_APIKEY='fake_apikey'
+# TRELLO_TOKEN='fake_trello_token' 
 
-TRELLO_TODO_ID='fake_todo_id'
-TRELLO_DOIN_ID='fake_doin_id'
-TRELLO_DONE_ID='fake_done_id'
\ No newline at end of file
+# TRELLO_TODO_ID='fake_todo_id'
+# TRELLO_DOIN_ID='fake_doin_id'
+# TRELLO_DONE_ID='fake_done_id'
+
+#docker doesnt like quotes(any)
+PYMONGO_CONN_STR=mongodb://fakemongo.com
+PYMONGO_TODO_ID='todo'
+PYMONGO_DOIN_ID='fake_doin_id'
+PYMONGO_DONE_ID='fake_done_id'
```

Comment:
It would be nice to also duplicate these values into the `.env.template` file so new devs can easily see what variables they need to set